src/tests/option_round/option_buyers/bidding_tests.cairo

```diff
@@ -276,15 +275,15 @@ fn test_bid_eth_transfer() {
     );
 
     // Check round balance
-    assert(round_balance_after == round_balance_before + bid_total, 'round balance after wrong');
+    assert(round_balance_after == round_balance_before + bids_total, 'round balance after wrong');
     // Check ob balances
     loop {
         match ob_balances_before.pop_front() {
             Option::Some(ob_balance_before) => {
-                let ob_bid_amount = bid_amounts.pop_front().unwrap();
+                let ob_bid_price = bid_prices.pop_front().unwrap();
                 let ob_balance_after = ob_balances_after.pop_front().unwrap();
                 assert(
-                    *ob_balance_after == *ob_balance_before - *ob_bid_amount,
+                    *ob_balance_after == *ob_balance_before - (*ob_bid_price * options_available),
```

Comment:
for test readability we should still pop off each bid amount (even tho it is options available for each) and use it here